Fix the compiling error that not include glib.h directly.

upstream bugzilla link: https://bugs.freedesktop.org/show_bug.cgi?id=52210
Upstream-Status: Submitted

Signed-off-by: Yao Zhao <yao.zhao@windriver.com>

diff --git a/test/core/test-service-glib.c b/test/core/test-service-glib.c
index 3355890..7291f35 100644
--- a/test/core/test-service-glib.c
+++ b/test/core/test-service-glib.c
@@ -11,7 +11,7 @@
 #include <string.h>
 #include <glib/gi18n.h>
 #include <glib-object.h>
-#include <glib/gquark.h>
+#include <glib.h>
 
 #include "my-object.h"
 #include "my-object-subclass.h"
